src/property-filter/__tests__/common.ts

```diff
@@ -59,9 +59,12 @@ export function toInternalProperties(properties: FilteringProperty[]): InternalF
   }));
 }
 
-export function toInternalOptions(options: FilteringOption[]): InternalFilteringOption[] {
+export function toInternalOptions(
```

Comment:
This is a logic duplication. What if the logic of this function deviate of the real one here: https://github.com/cloudscape-design/components/blob/70bafc50860ae991853c407766ca4d4a11199965/src/property-filter/index.tsx#L185-L190

Can you trust these tests after it?